heimdall.application/src/main/java/com/miqdigital/execution/dto/ExecutionInfo.java

```diff
@@ -9,8 +9,8 @@
 
 @Builder
 public class ExecutionInfo {
-  public String environment;
-  public String testType;
+  public String Environment;
```

Comment:
camelCase

Reply:
since it is public property that why it need to be not in camelCase